docs/source/user_guide/pipelines.md

```diff
@@ -14,141 +14,167 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 
-* Open the JupyterLab Launcher, if it is not open.
-* Click the `Pipeline Editor` icon to create an empty pipeline.
-  ![Main Page](../images/elyra-main-page.png)
-* From the sidebar open the JupyterLab File Browser.
-* Drag notebooks, Python or R scripts from the File Browser onto the canvas.
-* Define the dependencies between nodes by connecting them, essentially creating a graph.
-* Associate each node with a comment to document its purpose. 
+![The Visual Pipeline Editor is used to assemble pipelines](../images/user_guide/pipelines/pipeline-editor.png)
 
-  ![Pipeline Editor](../images/pipeline-editor-add-node.gif)
-* Right click on a node to open the associated file in the appropriate editor or define the node's runtime properties.
+Please review the [_Best practices for file-based pipeline nodes_ topic in the _User Guide_](best-practices-file-based-nodes.md) if your pipelines include generic components.
 
-  ![Access node properties](../images/node-context-menu.png)
-* Define the properties for each node in the pipeline.
+Elyra pipelines support three runtime platforms:
+- Local/JupyterLab
+- [Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/) (with Argo or [Tekton](https://github.com/kubeflow/kfp-tekton/) workflow engines)
+- [Apache Airflow](https://airflow.apache.org/)
 
-  ![Pipeline Node Properties](../images/pipeline-editor-properties.png)
+#### Generic pipelines
 
-  |Property   | Description  | Example |
-  |:---:|:------|:---:|
-  |Runtime Image| The container image you want to use to run your notebook |  `TensorFlow 2.0`   |
-  |File Dependencies|  A list of files to be passed from the `LOCAL` working environment into each respective step of the pipeline. Files should be in the same directory as the notebook it is associated with. Specify one file, directory, or expression per line. Supported patterns are `*` and `?`. | `dependent-script.py` |
-  |Environment Variables| A list of environment variables to be set inside in the container.  Specify one variable/value pair per line, separated by `=`. |  `TOKEN=value` |
-  |Output Files|  A list of files generated by the notebook inside the image to be passed as inputs to the next step of the pipeline.  Specify one file, directory, or expression per line. Supported patterns are `*` and `?`. | `data/*.csv` |
+A generic pipeline comprises only of nodes that are implemented using generic components.
+This Elyra release includes three generic components that allow for execution of Jupyter notebooks, Python scripts, and R scripts. 
 
-### Running a pipeline
+![Generic pipeline](../images/user_guide/pipelines/generic-pipeline.png)
 
-Pipelines run in your local JupyterLab environment or on Kubeflow Pipelines. 
+Generic pipelines are portable, meaning they can run locally in JupyterLab, or remotely on Kubeflow Pipelines or Apache Airflow.
 
-#### Running a pipeline in JupyterLab
+#### Typed pipelines
```

Comment:
Could we call this Runtime Specific pipeline?

Reply:
In the section heading or in general?

Reply:
Changed the terminology throughout the document from `typed pipeline` to `runtime-specific pipeline`